GoogleUtilities/Logger/Public/GoogleUtilities/GULLogger.h

```diff
@@ -102,66 +90,6 @@ extern void GULOSLogBasic(GULLoggerLevel level,
- * (required) log level (one of the GULLoggerLevel enum values).
- * (required) service name of type GULLoggerService.
- * (required) message code starting with "I-" which means iOS, followed by a capitalized
- *            three-character service identifier and a six digit integer message ID that is unique
- *            within the service.
- *            An example of the message code is @"I-COR000001".
- * (required) message string which can be a format string.
- * (optional) variable arguments list obtained from calling va_start, used when message is a format
- *            string.
- *
- * DEPRECATED; replaced by `GULOSLogBasic`.
- */
-extern void GULLogBasic(GULLoggerLevel level,
-                        GULLoggerService service,
-                        BOOL forceLog,
-                        NSString *messageCode,
-                        NSString *message,
-// On 64-bit simulators, va_list is not a pointer, so cannot be marked nullable
-// See: http://stackoverflow.com/q/29095469
-#if __LP64__ && TARGET_OS_SIMULATOR || TARGET_OS_OSX
-                        va_list args_ptr
-#else
-                        va_list _Nullable args_ptr
-#endif
-);
-
-/**
- * The following functions accept the following parameters in order:
- * (required) service name of type GULLoggerService.
- * (required) message code starting from "I-" which means iOS, followed by a capitalized
- *            three-character service identifier and a six digit integer message ID that is unique
- *            within the service.
- *            An example of the message code is @"I-COR000001".
- *            See go/firebase-log-proposal for details.
- * (required) message string which can be a format string.
- * (optional) the list of arguments to substitute into the format string.
- * Example usage:
- * GULLogError(kGULLoggerCore, @"I-COR000001", @"Configuration of %@ failed.", app.name);
- *
- * DEPRECATED; replaced by `GULOSLogError`, `GULOSLogWarning`, etc.
- */
-extern void GULLogError(
-    GULLoggerService service, BOOL force, NSString *messageCode, NSString *message, ...)
-    NS_FORMAT_FUNCTION(4, 5) DEPRECATED_MSG_ATTRIBUTE("Replaced by `GULOSLogError`.");
-extern void GULLogWarning(
-    GULLoggerService service, BOOL force, NSString *messageCode, NSString *message, ...)
-    NS_FORMAT_FUNCTION(4, 5) DEPRECATED_MSG_ATTRIBUTE("Replaced by `GULOSLogWarning`.");
-extern void GULLogNotice(
-    GULLoggerService service, BOOL force, NSString *messageCode, NSString *message, ...)
-    NS_FORMAT_FUNCTION(4, 5) DEPRECATED_MSG_ATTRIBUTE("Replaced by `GULOSLogInfo`.");
-extern void GULLogInfo(
-    GULLoggerService service, BOOL force, NSString *messageCode, NSString *message, ...)
-    NS_FORMAT_FUNCTION(4, 5) DEPRECATED_MSG_ATTRIBUTE("Replaced by `GULOSLogInfo`.");
-extern void GULLogDebug(
-    GULLoggerService service, BOOL force, NSString *messageCode, NSString *message, ...)
-    NS_FORMAT_FUNCTION(4, 5) DEPRECATED_MSG_ATTRIBUTE("Replaced by `GULOSLogDebug`.");
```

Comment:
Just as a reminder, when you move these functions to a new file internally, these were the ones I forget to remove the `DEPRECATED_MSG_ATTRIBUTE` from.